compose-network/prometheus/prometheus.yml

```diff
@@ -2,6 +2,13 @@ global:
   scrape_interval: 3s
 
 scrape_configs:
+  - job_name: 'relay-development'
+    scrape_interval: 30s
+    static_configs:
+    - targets: [ 'json-rpc-relay:7546' ]
```

Comment:
Q: Should this capture the `/metrics` path?

Reply:
Yes, it captures that path by default, no need to specify the `/metrics` path explicitly